src/app/cli.cljs

```diff
@@ -5,34 +5,72 @@
             ["child_process" :as cp]
             [clojure.string :as string]
             [cljs.reader :refer [read-string]]
-            [clojure.string :as string]))
+            [clojure.string :as string]
+            [cljs.core.async :refer [put! chan <! >! timeout close!]])
+  (:require-macros [cljs.core.async.macros :refer [go go-loop]]))
```

Comment:
This is also not required anymore. Just do

```
[cljs.core.async :refer [put! chan <! >! timeout close! go go-loop]
```

and remove the `:require-macros`.

Reply:
Glad to know!

I would need a better guide than this http://ku1ik.com/2015/10/12/sweet-core-async.html

Reply:
Well `core.async` is a bit too complex to be fully distilled into one single guide. It takes a bit of time to learn but it well worth the investment. I can absolutely recommend watching https://www.youtube.com/watch?v=096pIlA3GDo but there are other well written guides as well. Just don't have any links handy.

Note that it totally fine to still the the `:require-macros`, I just wanted to point out that it is not required anymore.